scripts/convert_toetsingskader_v1.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
its better to use #!/usr/bin/env bash


```suggestion
#!/usr/bin/env bash
```